utils/doinstall.tmplt

```diff
@@ -157,6 +157,22 @@ check_env
 # is defined
 unset CDPATH
 
+if [ $# = 1 ]; then
```

Comment:
it's cleaner to use arithmetic comparison rather than string comparison in this context since we know $# is a number:
if [ $# -eq 1 ]; then

Reply:
Thanks! Should be fixed now.